### PR TITLE
Fix/zero capacity nodes 5 x

### PR DESCRIPTION
**Pull request by ikalachy**
Motivation:

Explain here the context, and why you're making that change, what is the problem you're trying to solve.

Conformance:

Your commits should be signed and you should have signed the Eclipse Contributor Agreement as explained in https://github.com/eclipse/vert.x/blob/master/CONTRIBUTING.md
Please also make sure you adhere to the code style guidelines: https://github.com/vert-x3/wiki/wiki/Vert.x-code-style-guidelines

### CODE REVIEW

**Review by tsegismont — COMMENTED** [review]
Thanks for this PR @ikalachy 

Regarding tests I can't see where the Vert.x nodes are declared as zero capacity nodes. Can you clarify?

Also, I think it would be best to implement integration tests for zero capacity nodes like we have for `vertx-hazelcast`, see https://github.com/vert-x3/vertx-hazelcast/tree/master/vertx-hazelcast/src/test/java/io/vertx/spi/cluster/hazelcast/it/litemembers

File: vertx-infinispan/src/main/java/io/vertx/ext/cluster/infinispan/InfinispanClusterManager.java [hunk 1]
```diff
@@ -322,7 +323,8 @@ public String clusterPublicHost() {
   }
 
   private String getHostFromTransportProtocol(String fieldName) {
-    JGroupsTransport transport = (JGroupsTransport) cacheManager.getTransport();
+    JGroupsTransport transport = (JGroupsTransport) cacheManager.getCacheManagerConfiguration().transport().transport();
```
[the remark this answers is not in the document]
Comment (ikalachy) [reply vertx-infinispan/src/main/java/io/vertx/ext/cluster/infinispan/InfinispanClusterManager.java:326]: it's relevant for infinispan 15. May bad I was playing with version 15 locally so we can rollback.
